packages/cli/src/index.ts

```diff
@@ -428,4 +429,5 @@ yargs
   )
   .completion('completion', 'Generate completion script.')
   .demandCommand(1)
-  .strict().argv;
+  .strict()
+  .middleware([notifyUpdateCliVersion]).argv;
```

Comment:
Does it run before or after the command? Maybe we should run it after so not to slow down the commands?